app/src/main/java/com/example/myapplication/MatchData.java

```diff
@@ -59,28 +59,46 @@ public String returnAllData() {
     //Getters and Setters
 
     //Pre Match
-    public String getTeamNumber() {    return teamNumber;    }
-    public void setTeamNumber(String teamNumber) { this.teamNumber = teamNumber;    }
-    public String getMatchNumber() {    return matchNumber;    }
-    public void setMatchNumber(String matchNumber) { this.matchNumber = matchNumber;    }
+    public String getTeamNumber() {    
+        return teamNumber;    
+    
+    }
+    public void setTeamNumber(String teamNumber) { 
+        this.teamNumber = teamNumber;   
+    }
+    
+    public String getMatchNumber() {    
+        return matchNumber;    
+    }
+
+    public void setMatchNumber(String matchNumber) { 
+        this.matchNumber = matchNumber;    
+    }
```

Comment:
TeamNumber should be later stored as integer if there are no teams with leading zeroes. MatchNumber should later be stored as integer no matter what. ;)

Reply:
In the case of its usage within the scouting app, this technically isn't the case since we're only using this to match the data with the team and to place it as text on the screen, and not for any numeric operations so there aren't issues with team number being stored as a string